.github/workflows/pull_request.yml

```diff
@@ -51,7 +51,8 @@ jobs:
       matrix:
         pulp:
           - "3.16"
-          - "3.18"
+          - "3.21"
+          - "latest"
```

Comment:
Latest is a bit risky since it could break us

Reply:
Fair enough - removed